src/sensitivity.jl

```diff
@@ -42,9 +53,29 @@ macro explicit_intercepts(
     is_node::Expr=Expr(:vect, (true for _ in get_types(get_body(type_tuple)))...),
     kwargs::Expr=:(()),
 )
-    return esc(
-        explicit_intercepts(f, type_tuple, parse_is_node(is_node); parse_kwargs(kwargs)...)
-    )
+    isnode = parse_is_node(is_node)
+    kws = kwsort!(parse_kwargs(kwargs))
+    if isempty(kws)
+        def = nothing
+    else
+        newf = kwfname(f)
+        oldcall = Expr(:call, f)
+        newcall = Expr(:call, newf)
+        for i = 2:length(type_tuple.args)
+            arg = Symbol("arg", i - 1)
+            push!(newcall.args, Expr(:(::), arg, type_tuple.args[i]))
+            push!(oldcall.args, arg)
+        end
+        params = Expr(:parameters)
+        for (k, v) in pairs(kws)
+            push!(params.args, Expr(:kw, k, k))
+            push!(newcall.args, Expr(:kw, k, v))
+        end
+        insert!(oldcall.args, 2, params)
+        def = Expr(:function, newcall, oldcall)
+    end
+    ex = explicit_intercepts(f, type_tuple, isnode; kws...)
+    return esc(Expr(:block, def, ex))
```

Comment:
The above hunk needs some comments

Reply:
Done, let me know if they seem sufficient.